### PR TITLE
Move package release producer to stage

### PR DESCRIPTION
**Pull request by pacospace**
Signed-off-by: Francesco Murdaca <fmurdaca@redhat.com>

## Related Issues and Dependencies

Fixes: https://github.com/thoth-station/thoth-application/issues/422

## This introduces a breaking change

- [ ] Yes
- [x] No

## Test or Stage Environment

- [x] Pull Requests added content to STAGE environment

## This Pull Request implements

Thoth uses package-release Kafka producer in Stage

### CODE REVIEW

**Review by harshad16 — APPROVED** [review]
lgtm thanks :rocket:

File: package-releases/overlays/test/cronjob.yaml [hunk 1]
```diff
@@ -3,104 +3,8 @@ kind: CronJob
 apiVersion: batch/v1beta1
 metadata:
   name: package-releases
+  labels:
+    app: thoth
+    component: package-releases
```
Comment (goern) [inline package-releases/overlays/test/cronjob.yaml:8]: We don't want to have this as a disabled cronjob in test?

Comment (pacospace) [reply package-releases/overlays/test/cronjob.yaml:8]: I set `suspend: true` for test

Comment (harshad16) [reply package-releases/overlays/test/cronjob.yaml:8]: test is done, so we dont need it to executing in test.
if further testing is needed we can switch it back on.

**Pull request merged**